webclient/src/views/view/view-view.inc

```diff
@@ -239,31 +239,36 @@
           <div id="interactive-map" class="loading"></div>
         </div>
       </div>
-      <div
-        class="roomfinder-map-container"
-        v-bind:class="{'d-none': state.map.selected !== 'roomfinder'}"
-        v-if="view_data.maps.roomfinder && view_data.maps.roomfinder.available"
-      >
-        <img
-          alt="Cross showing where the room is located on the hand-drawn roomfinder map image"
-          src="<!-- @echo app_prefix -->assets/roomfinder_cross-v2.webp"
-          v-bind:style="{'transform': 'translate(' + state.map.roomfinder.x + 'px, ' + state.map.roomfinder.y + 'px)'}"
-          id="roomfinder-map-cross"
-        />
-        <img
-          alt="Hand-drawn roomfinder map image"
-          v-bind:src="'<!-- @echo cdn_prefix -->maps/roomfinder/' + view_data.maps.roomfinder.available[state.map.roomfinder.selected_index].file"
-          class="img-responsive"
-          v-bind:width="state.map.roomfinder.width"
-          v-bind:height="state.map.roomfinder.height"
-          id="roomfinder-map-img"
-        />
-        <div>
-          ${{_.view_view.map.img_source}}$: {{
-          view_data.maps.roomfinder.available[state.map.roomfinder.selected_index].source
-          }}
-        </div>
-      </div>
+        <a
+          v-on:click="setTimeout(loadModalRoomfinderMap, 1000)"
```

Comment:
I tried to delay the function but the `setTimeout()` function is somehow not working. I want that the cross is at first not displayed and when the` loadModalRoomfinderMap`-function is called the display style of the cross will change to `block`.